src/ProcessManager.php

```diff
@@ -128,11 +129,32 @@ class ProcessManager
     protected $port = 8080;
 
     /**
-     * Whether the server is in the reload phase.
+     * Whether the server is in the restart phase.
      *
      * @var bool
      */
-    protected $inReload = false;
+    protected $inRestart = false;
+
+    /**
+     * The number of seconds to wait before force closing a worker during a reload.
+     *
+     * @var int
+     */
+    protected $reloadThreshold = 30;
```

Comment:
Timeout instead of threshold?

Reply:
@fiskie could you check this last comment?

Reply:
I've had time to change this now.